__tests__/app.test.js

```diff
@@ -6,81 +6,139 @@ const app = require("../app.js");
+  test("should return a status of 400 if not given a number and a msg of bad request", () => {
     return request(app)
-    .get("/api/articles/500697")
-    .expect(404)
-    .then(({body: {msg}}) => {
-        expect(msg).toBe('Not Found')
+      .get("/api/articles/notanumber")
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad Request");
+      });
+  });
+});
+describe("PATCH /api/articles/:article_id", () => {
+  test("should respond with a status code of 200 and the updated article", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .expect(200)
+      .send({ inc_votes: 20 })
+      .then(({ body: { article } }) => {
+        expect(article).toEqual({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z", //There is a utility class tranlating a time stamp into date
+          votes: 120,
+        });
+      });
+  });
+  test('should respond with a smaller number of voes if passed a negative number to increment', () => {
+      return request(app)
+      .patch("/api/articles/1")
+      .expect(200)
+      .send({ inc_votes: -10 })
+      .then(({ body: { article } }) => {
+        expect(article).toEqual({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z", //There is a utility class tranlating a time stamp into date
+          votes: 90,
+        });
+      });
+  });
+  test('Should respond with a 400 error if given a request body with incorrect formatting of inc_votes', () => {
+    return request(app)
+    .patch("/api/articles/1")
+    .expect(400)
+    .send({ wrongName: 10 })
+    .then(({ body: { msg } }) => {
+         expect(msg).toBe('Bad Request')
+    })
+  });
+  test('should respond with a 400 error if given an object with too many properties', () => {
+    return request(app)
+    .patch("/api/articles/1")
+    .expect(400)
+    .send({ wrongName: 'hello' })
+    .then(({ body: { msg } }) => {
+         expect(msg).toBe('Bad Request')
     })
   });
-  test('should return a status of 400 if not given a number and a msg of bad request', () => {
+  test('should respond with a 400 error if given an object with the inc_votes property but with the incorrect data type', () => {
     return request(app)
-    .get("/api/articles/notanumber")
+    .patch("/api/articles/1")
     .expect(400)
-    .then(({body: {msg}}) => {
-        expect(msg).toBe('Bad Request')
+    .send({ wrongName: 10, shouldntbehere :10 })
+    .then(({ body: { msg } }) => {
+         expect(msg).toBe('Bad Request')
     })
   });
 });
```

Comment:
How would you want your server to respond if there were issues with the article id in the path or if no article could be found?